.gitmodules

```diff
@@ -3,7 +3,7 @@
 	url = https://github.com/Lukas-W/qt5-x11embed.git
 [submodule "plugins/ZynAddSubFx/zynaddsubfx"]
 	path = plugins/ZynAddSubFx/zynaddsubfx
-	url = https://github.com/lmms/zynaddsubfx.git
```

Comment:
The URL should not have changed at all, right?

Reply:
It should have, I set it to `https://github.com/bratpeki/zynaddsubfx.git` previously:

> I ended up using `git submodule set-url plugins/ZynAddSubFx/zynaddsubfx https://github.com/bratpeki/zynaddsubfx`. This has updated `.gitmodules`. Is this what you had in mind? If not, I will make further changes.

Reply:
I mean, compared to master this should not have changed. You can `git reset master -- .gitmodules` and then commit/push.

Reply:
On it!

Reply:
```
[peki@~/Code/lmms]$ git reset master -- .gitmodules
[peki@~/Code/lmms]$ git add .
[peki@~/Code/lmms]$ git diff
[peki@~/Code/lmms]$ git commit -m "Ran the command"
On branch master
Your branch is up to date with 'origin/master'.

nothing to commit, working tree clean
```

Seems as if, due to the command not changing anything now, it doesn't let the commit be made.

Reply:
Changing it manually right now

Reply:
Manually is also good :+1: